boards/nordic/nrf5340dk/board.yml

```diff
@@ -6,3 +6,9 @@ board:
     variants:
     - name: 'ns'
       cpucluster: 'cpuapp'
+  revision:
+    format: major.minor.patch
+    default: "0.14.0"
+    revisions:
+    - name: "0.7.0"
+    - name: "0.14.0"
```

Comment:
Unintentional change, I guess.

Reply:
ooops, danger of task switching.
Fixed.